.github/workflows/Pipeline.yml

```diff
@@ -17,11 +17,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - {python: '3.6', icon: 🔴}   # until 23.12.2021
-          - {python: '3.7', icon: 🟠}  # until 27.06.2023
-          - {python: '3.8', icon: 🟡}  # until Oct. 2024
-          - {python: '3.9', icon: 🟢}  # until Oct. 2025
-          - {python: '3.10', icon: 🟢} # until Oct. 2026
+          - {python: "3.6",  icon: 🔴}   # until 23.12.2021
```

Comment:
There is an extra space before `#`.

Reply:
Oh I got it. It's a problem in PyCharm. The font renders a red dot in different size, so I added one more space to align the comments ...

![image](https://user-images.githubusercontent.com/956109/143492452-798f0036-f327-4c05-8804-fb67b32c6921.png)

Reply:
@umarcor fixed it. please review again.